### PR TITLE
refactor: 💡 抽离hook 增加alias

### PR DESCRIPTION
**Pull request by kibuniverse**

### CODE REVIEW

File: src/hooks/useScroll.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const useControlHeader = (initOpacity: boolean) => {
+	const [scrollerHeight, setScrollerHeight] = React.useState(0)
+	const [hidden, setHidden] = React.useState(false)
+	const [opacity, setOpacity] = React.useState(initOpacity)
+
+	React.useEffect(() => {
+		const fn = () => {
+			setScrollerHeight((item) => {
+				if (item > document.documentElement.scrollTop) {
```
Comment (lovebaihezi) [inline src/hooks/useScroll.tsx:11]: setHidden(item <= document.documentElement.scrollTop)

**Review by lovebaihezi — CHANGES_REQUESTED** [review]
need test

File: src/hooks/useScroll.tsx (hunk 1 above)
Comment (kibuniverse) [reply src/hooks/useScroll.tsx:11]: @lwsymyself

**Review by liuyuting220 — APPROVED** [review]

**Review by lovebaihezi — COMMENTED** [review]
need change

**Review by lovebaihezi — APPROVED** [review]

**Comment by lovebaihezi** [comment]
[update] hooks eslint

**Pull request merged**